qlc/qlc.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why exist two sub-folder, qlc and Qlc, just different capital letter is upper and lower?

Reply:
That's a very good question - it does not look like that on disk

Reply:
Fixed it - apparently it was related to this: https://github.com/gitextensions/gitextensions/issues/1614